src/plugins/orangehrmPerformancePlugin/config/routes.yaml

```
@@ -12,6 +12,34 @@ apiv2_performance_kpis:
   defaults:
     _api: OrangeHRM\Performance\Api\KpiAPI
 
+apiv2_performance_supervisors:
+  path: /api/v2/performance/supervisors
```

```suggestion
  path: /api/v2/performance/reviews/supervisors
```
:thinking: how about this